iib/workers/tasks/build_regenerate_bundle.py

```diff
@@ -98,13 +101,21 @@ def handle_regenerate_bundle_request(
         # Pull the from_bundle_image to ensure steps later on don't fail due to registry timeouts
         podman_pull(from_bundle_image_resolved)
 
+        if bundle_replacements:
+            log.debug(f'Ignoring the following bundle_replacements: {bundle_replacements}')
+
         with tempfile.TemporaryDirectory(prefix='iib-') as temp_dir:
             manifests_path = os.path.join(temp_dir, 'manifests')
             _copy_files_from_image(from_bundle_image_resolved, '/manifests', manifests_path)
             metadata_path = os.path.join(temp_dir, 'metadata')
             _copy_files_from_image(from_bundle_image_resolved, '/metadata', metadata_path)
             new_labels = _adjust_operator_bundle(
-                manifests_path, metadata_path, request_id, organization, pinned_by_iib
+                manifests_path,
+                metadata_path,
+                request_id,
+                organization,
+                pinned_by_iib,
+                bundle_replacements,
```

Comment:
Let's make this call parameterized? `bundle_replacements=bundle_replacements `

Reply:
The current call is passing bundle_replacements in `recursive_related_bundles`. Could you please parameterize this call for optional variables?